llvm/lib/Target/AIE/AIEMachineScheduler.h

```diff
@@ -190,6 +196,8 @@ class AIEPreRASchedStrategy : public GenericScheduler {
   std::vector<unsigned> SUDelayerMap;
 
   std::vector<unsigned> PSetThresholds;
+
+  AIEAlternateDescriptors SelectedAltDescs;
```

Comment:
We should probably clear that on `enterRegion`, otherwise the map will grow un-necessarily.

Reply:
I have done this in `leaveRegion`